MAUI/ImageEditor/migration.md

```diff
@@ -527,4 +527,5 @@ Applicable for crop view selection, shape and text annotation selection.
 
 ## Known issues in .Net MAUI ImageEditor
 * [Mac, iOS]The image is not loading correctly from the stream or URI binding source.
-* Rotation not working for text annotation.
\ No newline at end of file
+* Rotation not working for text annotation.
+* [Mac, iOS]FontAttributes will not work for text annotation with Custom font family.
```

Comment:
* [Mac, iOS] The FontAttributes will not work for text annotations with the Custom font family.